[hsa-rocr] Build failed in clean chroot

Build log:
```
==> Starting build()...
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found LibElf: /usr/lib/libelf.so  
-- Performing Test ELF_GETSHDRSTRNDX
-- Performing Test ELF_GETSHDRSTRNDX - Success
-- Looking for __NR_memfd_create
-- Looking for __NR_memfd_create - found
CMake Error at image/blit_src/CMakeLists.txt:49 (find_package):
  Could not find a package configuration file provided by "Clang" with any of
  the following names:

    ClangConfig.cmake
    clang-config.cmake

  Add the installation prefix of "Clang" to CMAKE_PREFIX_PATH or set
  "Clang_DIR" to a directory containing one of the above files.  If "Clang"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "/build/hsa-rocr/src/CMakeFiles/CMakeOutput.log".
==> ERROR: A failure occurred in build().
```

--- a/hsa-rocr/.SRCINFO
+++ b/hsa-rocr/.SRCINFO
@@ -1,14 +1,16 @@
 pkgbase = hsa-rocr
 	pkgdesc = ROCm Platform Runtime: ROCr a HPC market enhanced HSA based runtime
 	pkgver = 3.7.0
-	pkgrel = 2
+	pkgrel = 3
 	url = https://rocmdocs.amd.com/en/latest/Installation_Guide/ROCR-Runtime.html
 	arch = x86_64
 	license = custom:NCSAOSL
 	makedepends = cmake
+	makedepends = llvm-amdgpu
+	makedepends = xxd
 	depends = libelf
 	depends = hsakmt-roct
-	depends = xxd
+	depends = rocm-device-libs
 	provides = rocr-runtime=3.7.0
 	conflicts = rocr-runtime
 	replaces = rocr-runtime

--- a/hsa-rocr/PKGBUILD
+++ b/hsa-rocr/PKGBUILD
@@ -6,13 +6,13 @@
 
 pkgname=hsa-rocr
 pkgver=3.7.0
-pkgrel=2
+pkgrel=3
 pkgdesc='ROCm Platform Runtime: ROCr a HPC market enhanced HSA based runtime'
 arch=('x86_64')
 url='https://rocmdocs.amd.com/en/latest/Installation_Guide/ROCR-Runtime.html'
 license=('custom:NCSAOSL')
-depends=('libelf' 'hsakmt-roct' 'xxd')
-makedepends=('cmake')
+depends=('libelf' 'hsakmt-roct' 'rocm-device-libs')
+makedepends=('cmake' 'llvm-amdgpu' 'xxd')
 provides=("rocr-runtime=$pkgver")
 replaces=('rocr-runtime')
 conflicts=('rocr-runtime')

